pkg/cloud/provider/csvprovider.go

```diff
@@ -279,7 +279,7 @@ func (c *CSVProvider) NodePricing(key models.Key) (*models.Node, models.PricingM
 			}
 		}
 		totalCost := hourly * float64(count)
-		node.GPUCost = fmt.Sprintf("%f", totalCost)
+		node.GPUCost = fmt.Sprintf("%f", hourly)
```

Comment:
If this has to be modified, doesn't the line a little below 
```
node.Cost = fmt.Sprintf("%f", nc+totalCost)
```

have to be modified?

Reply:
Also, I don't understand why this isn't `totalCost` -- if there are multiple GPUs, shouldn't the cost be multiplied? Or is this because `GPUClassPricing` contains prices that total the GPUs on a node, rather than breaking them out individually?

Reply:
This is the field that gets transformed into node_gpu_hourly_cost . Its sister queries are node_cpu_hourly_cost and node_ram_hourly_cost which are unit prices per 1cpu and 1GB RAM.